.gitignore

```diff
@@ -1,24 +1,14 @@
-# Don't put *.swp, *.bak, etc here; those belong in a global ~/.gitignore.
+# Don't put *.swp, *.bak, etc here; those belong in a global ~/.gitignore_global.
```

Comment:
The default global ignore is `$XDG_CONFIG_HOME/git/ignore`. If `$XDG_CONFIG_HOME` is either not set or empty, `$HOME/.config/git/ignore` is used instead. ([source](https://git-scm.com/docs/gitignore/)) This should either be updated to reflect that, or left as is, but not changed to some arbitrary value.